engine/lib/output.php

```diff
@@ -391,11 +391,21 @@ function elgg_get_friendly_time($time, $current_time = null) {
 	$minute = 60;
 	$hour = $minute * 60;
 	$day = $hour * 24;
+	$month = $day * 30;
+	$year = $day * 365;
 
 	if ($diff < $minute) {
-		return elgg_echo("friendlytime:justnow");
+		return elgg_echo('friendlytime:justnow');
 	}
 	
+	if ($diff > $year) {
+		return trim(elgg_echo('date:month:' . date('m', $time), [''])) . ' ' . date('Y', $time);
```

Comment:
I don't think all languages want to use the same order.

Something like this would allow parameter swapping:
```php
$month = date('m', $time);
$month = elgg_echo("date:month:$month");
$year = date('Y', $time);

return trim(elgg_echo('date:month_and_year', [$month, $year]));
```
Not even this is probably good enough for all languages. But an improvement in any case.